shivakrishna/facebook_login.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#3b5998"
+    tools:context="com.example.root.facebook.MainActivity">
+
+<ImageView
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:src="@drawable/fblogo"
+    android:layout_marginTop="70dp"
+    android:id="@+id/logo"/>
+    <TextView
```

Comment:
Sorry for the silly mistakes! I am in a hurry to submit the assignment. I was unable to test it on my phone. It says builds are failing. Will submit it again.